src/python/s2_common.i

```diff
@@ -372,6 +498,9 @@ class S2Point {
 
 %ignoreall
 
+%unignore MutableS2ShapeIndex;
+%unignore MutableS2ShapeIndex::~MutableS2ShapeIndex;
+%unignore MutableS2ShapeIndex::Add(S2Shape*);
```

Comment:
There are a lot of functions added. Are they all tested? There are too many to easily tell.

Reply:
I hope so, but I'll review the changes to make sure. (I'll leave this open for now to remind me to do it.)

Reply:
Oh, I realised that I didn't add tests for the ones that were basically just a `%unignore` - is this what you meant? I can add these if you like.

Reply:
I think everything is covered now.